packages/atomic/src/components/facets/atomic-timeframe-facet/atomic-timeframe-facet.tsx

```diff
@@ -231,7 +231,11 @@ export class AtomicTimeframeFacet
 
   private formatFacetValue(facetValue: DateFacetValue) {
     try {
-      const relativeDate = deserializeRelativeDate(facetValue.start);
+      const startDate = deserializeRelativeDate(facetValue.start);
+      const relativeDate =
+        startDate.period === 'past'
+          ? startDate
+          : deserializeRelativeDate(facetValue.end);
```

Comment:
Could you explain these lines? If the period is not in the `past`, we use the `end` date?

Is there an assumption that the dates are always relative to `now`?

Reply:
The dates can only be the following options (using atomic-timeframe, they are always relative to now):
- now -> next
- past -> now

The relativeDate cannot be now, it is either past or next

Reply:
<img width="479" alt="Screen Shot 2021-09-27 at 12 04 17 PM" src="https://user-images.githubusercontent.com/4923043/134944711-d46c73fa-5f12-4f3b-a4fc-3330a94e5c22.png">